intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -172,6 +172,7 @@ The documentation tells us that `log()` is derived from `{base}`, meaning it is
 It provides a brief description of what the function does and shows several examples of to how use it.
 
 In particular, documentation tells about how what argument(s) to provide:
+
```

Comment:
Noting this new line is needed since bullets aren't rendering as bullets without it.